app/assets/stylesheets/application.css

```diff
@@ -4,40 +4,50 @@
 :root {
 	color-scheme: light dark;
 
-	/* main monochrome */
-	--palette-bg: light-dark(rgb(254, 254, 254), rgb(12, 12, 12));
-	--palette-fg: light-dark(rgb(51, 51, 51), rgb(225, 225, 225));
-	--palette-light: rgb(255, 255, 255);
-	--palette-shadow: rgb(0, 0, 0);
-
-	/* blends of foreground color over the background */
-	--opacity-contrast-15: light-dark(100%, 87%); /* Contrast capped at 10:1 for dark mode */
-	--opacity-contrast-13: light-dark(94%, 87%);
-	--opacity-contrast-10: light-dark(84%, 87%);
-	--opacity-contrast-7-5: light-dark(75%, 77%);
-	--opacity-contrast-6: light-dark(67%, 69%);
-	--opacity-contrast-5: light-dark(63%, 64%);
-	--opacity-contrast-4-5: light-dark(59%, 61%); /* WCAG AA minimum 4.5:1 text contrast */
-
-	--opacity-gradient-lit: light-dark(0%, 13%);
-	--opacity-gradient-shadowed: light-dark(13%, 3%);
+	/** main monochrome and variants **/
+	--base-bg-light: 254 254 254;
```

Comment:
Why put this in a variable instead of directly in the `--color-bg` a few lines down? I don't see that it's reused.